common.blocks/checkbox/checkbox.js

```diff
@@ -12,13 +12,18 @@ modules.define('checkbox', ['i-bem__dom', 'control'], function(provide, BEMDOM,
  */
 provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends checkbox.prototype */{
     onSetMod : {
-        'checked' : function(modName, modVal) {
-            this.elem('control').prop(modName, modVal);
+        'checked' : {
+            'true' : function() {
+                this.elem('control').attr('checked', true);
+            },
+            '' : function() {
+                this.elem('control').removeAttr('checked');
+            }
         }
     },
 
     _onChange : function() {
-        this.setMod('checked', this.elem('control').prop('checked'));
+        this.toggleMod('checked');
```

Comment:
TODO set mod explicitly
`setMod('checked', !!this.elem('control').attr('checked'));`